kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -810,7 +816,11 @@ def test_pod_template_file_with_full_pod_spec(self):
         assert k.pod.spec.containers[0].env == [k8s.V1EnvVar(name="env_name", value="value")]
         assert result == {"hello": "world"}
 
-    def test_full_pod_spec(self):
+    @pytest.mark.parametrize(
```

Comment:
Adding support for this required me to move from a `unittest.TestCase` to pytest. This required a couple of other fixes, but is worth it in my mind, since other test classes are using pytest.